compiler/compile/OMRCompilation.cpp

```diff
@@ -97,6 +97,7 @@
 #include "control/Recompilation.hpp"           // for TR_Recompilation, etc
 #include "runtime/CodeCacheExceptions.hpp"
 #include "ilgen/IlGen.hpp"                     // for TR_IlGenerator
+#include "env/CompileTimeProfiler.hpp"         // for TR::CompileTimeProfiler
```

Comment:
Would this build on non Linux platforms? I'm not sure `#include <sys/syscall.h>` exists on Windows, for example. You may want to guard this include with an `#if defined(LINUX)`.